docs/logs/correlation/Program.cs

```diff
@@ -17,6 +17,7 @@
 using System.Diagnostics;
 using Microsoft.Extensions.Logging;
 using OpenTelemetry;
+using OpenTelemetry.Trace;
```

Comment:
Although this is technically correct, I wonder if we could make it simpler by not requiring this namespace?